api/v1alpha1/packagebundle_types.go

```diff
@@ -83,6 +83,9 @@ type SourceVersion struct {
 
 	// Images is a list of images used by this version of the package
 	Images []VersionImages `json:"images,omitempty"`
+
+	// Configurations is a list of configurations used by this version of the package
```

Comment:
I wouldn't mind an added bit of commentary for how these are used.

Reply:
Done